### PR TITLE
Ability to retrieve username and password from the environment

### PR DESCRIPTION
**Pull request by joliver**
While CLI arguments work, the username and password values are visible to anyone that can get a process list. Further, some control planes (Kubernetes, Mesos, among others), log the CLI arguments to stderr which would then include the UpCloud username and password.

This PR makes it possible to specify those values via environment variables if they aren't provided in the CLI.

### CODE REVIEW

**Review by nate-droid — CHANGES_REQUESTED** [review]
Hi! 
This is a great change and would provide better functionality! Thanks for taking the time to put this together.

Before having this merged, would you be able to update the variable names to `UPCLOUD_USERNAME` and `UPCLOUD_PASSWORD` so that they would be consistent with the naming conventions we use in our other projects (For example the Go SDK)

Thanks!

**Comment by joliver** [comment]
The change has been pushed.

**Comment by nate-droid** [comment]
Thanks! I've just approved, and will have it merged once the pipelines run 😄

**Review by nate-droid — APPROVED** [review]

**Pull request merged**
